streams/src/test/java/org/apache/kafka/streams/processor/internals/HighAvailabilityStreamsPartitionAssignorTest.java

```diff
@@ -148,34 +155,27 @@ private void createMockTaskManager(final Set<TaskId> activeTasks) {
     }
 
     private void createMockTaskManager(final Map<TaskId, Long> taskOffsetSums) {
-        taskManager = EasyMock.createNiceMock(TaskManager.class);
-        expect(taskManager.topologyMetadata()).andStubReturn(topologyMetadata);
-        expect(taskManager.getTaskOffsetSums()).andReturn(taskOffsetSums).anyTimes();
```

Comment:
Why was this stub not migrated?

Reply:
Mockito said that the stubbing was unnecessary. However, now that you have asked I will look further into the test.

Reply:
Yeah, after looking a bit more into this it just seems that this code path is not exercised. I deleted it from the original EasyMock tests and the tests still passed, so I think it was not exercised in the original test either. I put a breakpoint in the beginning of the test and it was never reached.